database/pgcsv_to_json

```diff
@@ -42,13 +42,13 @@ def convert_type_func(ty, ty_rest = ""):
         return re.sub(r"\\(.)", lambda m: '""' if m.group(1) is '"' else m.group(1), s)
     if ty_el == "text":
       def convert_text_array(value):
-        arr = csv.reader([backslashes_to_csv_escapes(value[1:-1])], delimiter=',', quotechar='"', escapechar='\\').next()
-        return map(convert, arr)
+        arr = next(csv.reader([backslashes_to_csv_escapes(value[1:-1])]))
```

Comment:
Looks like this `csv.reader` is seeing a newline here?  I think this `backslashes_to_csv_escapes` was depending on the assumption of arrays appearing in a TSV stream.  So if we switch the input to CSV, not sure if we can keep the rest intact.